website/docs/language/settings/backends/s3.html.md

```diff
@@ -199,7 +199,7 @@ The following configuration is optional:
 The following configuration is optional:
 
 * `dynamodb_endpoint` - (Optional) Custom endpoint for the AWS DynamoDB API. This can also be sourced from the `AWS_DYNAMODB_ENDPOINT` environment variable.
-* `dynamodb_table` - (Optional) Name of DynamoDB Table to use for state locking and consistency. The table must have a primary key named `LockID` with type of `string`. If not configured, state locking will be disabled.
+* `dynamodb_table` - (Optional) Name of DynamoDB Table to use for state locking and consistency. The table must have a Partition key named `LockID` with type of `String`. If not configured, state locking will be disabled.
```

Comment:
Thank you! Please just uncapitalize "Partition" - it's not in caps in DynamoDB docs :-)

Reply:
thanks @laurapacilio - turns out I cleaned up the source fork (housekeeping!), plus there were merge conflicts here - so have re-created.

https://github.com/hashicorp/terraform/pull/29494